everviz/plugins/crossplot/crossplot.py

```diff
@@ -18,6 +18,11 @@
 
 
 class Crossplot(WebvizPluginABC):
+    """
+    The Crossplot class implements a plugin for Webviz, which can plot any
+    column from the data exported by Everest against any other column.
```

Comment:
Technically it can plot any column against any other column in a `csv`, might want to clarify that? That is kind of the case with `CrossplotIndexed` as well, however that extracts columns with `*-x` in the column name, so is very specific. Because of that I think you suggestion works fine for that.

Reply:
> Technically it can plot any column against any other column in a `csv`, might want to clarify that? That is kind of the case with `CrossplotIndexed` as well, however that extracts columns with `*-x` in the column name, so is very specific. Because of that I think you suggestion works fine for that.

Indeed, I have amended the text.